pkg/support/dump_others_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !windows
+// +build !windows
+
+package support
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"antrea.io/antrea/pkg/util/logdir"
+
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDumpLog(t *testing.T) {
+	fs := afero.NewMemMapFs()
+	logDir := logdir.GetLogDir()
+
+	fs.MkdirAll(logDir, os.ModePerm)
+	fs.Create(filepath.Join(logDir, "antrea-agent.log"))
+	fs.Create(filepath.Join(logDir, "ovs.log"))
+	fs.Create(filepath.Join(logDir, "kubelet.log"))
+
+	dumper := NewAgentDumper(fs, nil, nil, nil, nil, "7s", true, true)
+	err := dumper.DumpLog(baseDir)
+	require.NoError(t, err)
+
+	x, err := afero.Exists(fs, filepath.Join(baseDir, "logs", "agent", "antrea-agent.log"))
```

Comment:
rename the `x` variables to `ok` (there is another test like this as well), which is more in line with Go conventions